crates/web-sys/webidls/enabled/AudioBufferSourceNode.webidl

```diff
@@ -32,7 +32,12 @@ interface AudioBufferSourceNode : AudioScheduledSourceNode {
     attribute double loopStart;
     attribute double loopEnd;
 
+                    attribute EventHandler onended;
+
     [Throws]
     void start(optional double when = 0, optional double grainOffset = 0,
                optional double grainDuration);
+
+    [Throws]
+    void stop (optional double when = 0);
```

Comment:
Note that this class didn't switch to `AudioBufferSourceNode includes rustAudioScheduledSourceNode;` because the mixin's `start` method conflicts with the `start` method in this class, so it was just easier to inline the few properties here.